packages/teams-js/src/public/meeting.ts

```diff
@@ -99,7 +99,25 @@ export namespace meeting {
    * Hide from docs
    * Data structure to represent call details
    */
-  export type ICallDetails = IMeetingOrCallDetailsBase<CallType>;
+  export interface ICallDetails extends IMeetingOrCallDetailsBase<CallType> {
+    /**
+     * @hidden
+     * Unique identifier for the original caller
+     */
+    originalCaller?: string;
+
+    /**
+     * @hidden
+     * Entity dialed by the caller
+     */
+    dialedEntity?: string;
+
+    /**
+     * @hidden
+     * Tracking identifier for grouping related calls
```

Comment:
We're still working with partners to define the specific format of the identifier. Once that format is determined, I will make sure to push a change to update this description.

Reply:
Are you expecting to have the format finalized before you want to commit this change to main? Or are you looking to "push a change" in a subsequent PR?

Reply:
We were planning to commit this interface to main as part of this PR first, while finalizing the format in parallel.

The properties `dialedEntity` & `trackingId` actually still need to be designed and implemented by a partner team, and we won't be returning these values as part of this initial version / PR (hence why the format is still under consideration).

I had a discussion with the Taos team previously, and our conclusion then was that, as these new parameters are optional, we aren't guaranteeing that the properties would be populated. As a result, we can include these pending properties in the interface while they're under development.

However, if the preference is for us to include those pending properties in a subsequent PR, I can also remove those properties from this PR and add them back once they're implemented.

Reply:
Since they are not yet decided, I presume that means that they can't practically be used yet, either. Is that right? If so but you want to keep them in here, could you mark them as type `unknown` (or even `never`) so that way no one is tempted to use them until their data shape is figured out?

Reply:
That makes sense! I have updated the types to `never`.